README.md

```diff
@@ -118,4 +118,23 @@ extra_hosts:
       - "host.docker.internal:host-gateway"
 ```
 
-IMPORTANT: Don't include this in a PR, because it will break stuff on macOS.
\ No newline at end of file
+IMPORTANT: Don't include this in a PR, because it will break stuff on macOS.
+
+
+
+## Debugging locally
+
+To save the arguments (`req` and `data`) to a worker function, specify DEBUG_STEP. DEBUG_STEP corresponds to the basename of the paths in work.r and will hot-reload if changed at the top of work.r
+
+```bash
+# e.g. DEBUG_STEP=getClusters
+DEBUG_STEP=task_name npm start
```

Comment:
actually now that the Makefile is merged, you can just do `make build && make run`

Reply:
Ah, thanks Pol, I've changed it, hope that's fine now (haven't tested it though)

Reply:
> Really helpful PR!
> Just a few comments/questions:
> 
> 1. `debug/{taskname}.rdata` is overwritten everytime a `taskname` is called, this can be quite taxing since it contains the large seurat object doesn't really change causing a lot of unnessary I/O and might cause hard to debug time-outs. You have pointed this already out, but I think this might become an issues for larger datasets.
>    A mitigation might be to save the seurat object seperatly as a file and only update the `req`. The downside being less ease of use (not everything is in one rdata).. I'd still prefer this one seperate option though.
> 2. If ones changes the experiment, the `./debug/{task}.rdata` object will be overwritten, I'd propose to have experimental ID as part of the filename `./debug/{EID}_{taskname}.rdata` or so.
> 3. As with pipeline, an `all` option would be very helpful (see below)
> 
> A possible strategy for all points could be to save the seurat object as `./debug/{EID}.rds` only if it doesn't exist already (which would save I/O). Alongside an `./debug/{EID}_{taskname}.rdata` only containing the `req` object, this will always be overwritten when taskname is called. The option `all` would than allow to generate multiple rdata objects for each task (which are quick to generate).

Thanks for the insightfull comments @seb-mueller. I totally agree and will make these changes.